ls1intum/Artemis
# [Exam] Bugfix tutor dashboard

### Checklist

### Motivation and Context


Fixed following bugs:
1. Allow tutors to access the exam management and exam exercise dashboard from the course management view if the exam due date is over
3. (Tutor) There is a permission problem with example submissions (to review / assess): Tutors cannot access the example submissions
4. (Instructor) When creating modeling exercises, there is something about text exercises ("[...] must finish creating the text exercise" or so)
5. (Instructor) The back buttons for creating the Example Solution and the Example Submissions during modeling do not work (-> console error)
6. (Instructor, as Tutor does not have access yet) The back buttons for leaving the example submissions (to review / assess) in the Tutor Exam Dashboard do not work (-> console error)

### Steps for Testing


1. Navigate to course management
2. click on exams 
3. add an exercise group with a modeling exercise
4. check that the bugs are fixed -> play around with creating example solution/submission
5. check that you don't get console error when navigating back from the example solution/submission page
6. navigate to the tutor dashboard and check if you have permission to access example submissions
-----------------------------
1. login as a tutor only
2. navigate to exam management -> check that you *only* have access to the tutor dashboard
3. access the dashboard and try access an example submission
